docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>This should also use the <code>ref</code> syntax, which to work will need you to edit that file and add the target at the top right before the title which seems to be missing. The target is the <code>(target)=</code> thing like you have added at the top of this file  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='114'/>

Reply:
+1 Added!